terraform/modules/vm_with_pf_rules_with_ssh_access/main.tf

```diff
@@ -0,0 +1,49 @@
+resource "openstack_compute_instance_v2" "instance_01" {
+  name = var.vm_name
+  flavor_name = var.flavor_name
+  key_pair = var.access_key
+
+  block_device {
+    uuid                  = var.image_id
+    source_type           = "image"
+    volume_size           = 20
```

Comment:
is this `volume_size` also hardcoded? what happens if an users request a bigger root fs from a flavor rather than `FLAVOR_NAME="CPUv1.small"
`?

Reply:
changed by commit 30484244c5f81368ba7bd7c09016153ddb0f4cd6
root fs volume size is now based on flavor disk size